### PR TITLE
VIH-6385 cvp audio file retrieval 

### PR DESCRIPTION
**Pull request by rehmanab**
https://tools.hmcts.net/jira/browse/VIH-6385

### CODE REVIEW

File: VideoAPI/VideoApi.Contract/Responses/CvpAudioRecordingResponse.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+using System.Collections.Generic;
+
+namespace VideoApi.Contract.Responses
+{
+    public class CvpAudioRecordingResponse
+    {
+        public IEnumerable<CvpAudioFile> Results { get; set; }    
+    }
```
Comment (valentinabennett) [inline VideoAPI/VideoApi.Contract/Responses/CvpAudioRecordingResponse.cs:8]: Why we cannot return just IEnumerable<CvpAudioFile> and not create additional class?

File: VideoAPI/VideoApi.Services/CvpAzureStorageService.cs [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+using System.Threading.Tasks;
+using Azure.Storage;
+using Azure.Storage.Blobs;
+using Azure.Storage.Sas;
+using VideoApi.Common.Configuration;
+using VideoApi.Services.Contracts;
+
+namespace VideoApi.Services
+{
+    public class CvpAzureStorageService : IAzureStorageService
```
Comment (shaed-parkar) [inline VideoAPI/VideoApi.Services/CvpAzureStorageService.cs:12]: The implementation looks pretty identical to `VhAzureStorageService`. Would suggest a base or any solution to to share the methods

**Review by shaed-parkar — COMMENTED** [review]
Storage service classes are identical, do need both?

File: VideoAPI/VideoApi.Services/CvpAzureStorageService.cs (hunk 2 above)
Comment (rehmanab) [reply VideoAPI/VideoApi.Services/CvpAzureStorageService.cs]: done

**Review by valentinabennett — APPROVED** [review]

**Pull request merged**